litmus-portal/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -228,6 +229,52 @@ func CreateChaosWorkflow(input *model.ChaosWorkFlowInput, r store.StateData) (*m
 	}, nil
 }
 
+func DeleteCluster(cluster_id string, r store.StateData) (string, error) {
+	time := strconv.FormatInt(time.Now().Unix(), 10)
+
+	query := bson.D{{"cluster_id", cluster_id}}
+	update := bson.D{{"$set", bson.D{{"is_removed", true}, {"updated_at", time}}}}
+
+	err := database.UpdateCluster(query, update)
+	if err != nil {
+		return "", err
+	}
+	cluster, err := database.GetCluster(cluster_id)
+	if err != nil {
+		return "", nil
+	}
+
+	requests := []string{
+		`{
```

Comment:
What about the other components?

Reply:
Only subscriber and config map needs to be deleted